integrations/oci/atp/src/main/java/io/helidon/integrations/oci/atp/GenerateAutonomousDatabaseWallet.java

```diff
@@ -85,15 +87,15 @@ public WalletArchive walletArchive() {
      * Object to store wallet returned for ATP as bytes[].
      */
     public static class WalletArchive {
-        private byte[] content;
+        private final byte[] content;
 
         /**
          * Set wallet data.
          *
          * @param content
          */
-        public WalletArchive(final byte[] content) {
-            this.content = content;
+        public WalletArchive(byte[] content) {
+            this.content = Arrays.copyOf(content,content.length);
```

Comment:
Or `this.content = content.clone()`

Reply:
I think the performance is the same, but using `clone()` was rather discouraged in  the Effective Java book

Reply:
IIRC, its more about custom object implementing clone method in book. I don't have preference over one vs other as, like Daniel said, performance is almost the same.

Reply:
Fixed.